qiita_db/processing_job.py

```diff
@@ -620,6 +620,29 @@ def _update_and_launch_children(self, mapping):
         for c in ready:
             c.submit()
 
+    @property
+    def outputs(self):
+        """The outputs of the job
+
+        Returns
+        -------
+        dict of {str: qiita_db.artifact.Artifact}
+            The outputs of the job keyed by output name
+        """
+        with qdb.sql_connection.TRN:
+            if self.status != 'success':
```

Comment:
Should this really raise an error? I think sending an empty array should suffice, no?